cuda/btx_cudamatching_model.yaml

```diff
@@ -0,0 +1,293 @@
+  - :set_id: ctx_destroy_exit
+    :domain: exits & ctx_destroy
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^cuResult$
+  
+  # Context stack management
+  - :set_id: ctx_set_current
+    :name: "cuCtxSetCurrent"
+    :register: false
+  - :set_id: ctx_set_current_entry
+    :domain: entries & ctx_set_current
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^ctx$
+  - :set_id: ctx_set_current_exit
+    :domain: exits & ctx_set_current
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^cuResult$
+  
+  - :set_id: ctx_push_current
+    :name: "cuCtxPushCurrent"
+    :register: false
+  - :set_id: ctx_push_current_entry
+    :domain: entries & ctx_push_current
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^ctx$
+  - :set_id: ctx_push_current_exit
+    :domain: exits & ctx_push_current
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^cuResult$
+  
+  - :set_id: ctx_pop_current
+    :name: "cuCtxPopCurrent"
+    :register: false
+  - :set_id: ctx_pop_current_entry
+    :domain: entries & ctx_pop_current
+  - :set_id: ctx_pop_current_exit
+    :domain: exits & ctx_pop_current
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^cuResult$
+      - :name: ^pctx_val$
+  
+  # Device primary context management
+  - :set_id: primary_ctx_retain
+    :name: "cuDevicePrimaryCtxRetain"
+    :register: false
+  - :set_id: primary_ctx_retain_entry
+    :domain: entries & primary_ctx_retain
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^dev$
+  - :set_id: primary_ctx_retain_exit
+    :domain: exits & primary_ctx_retain
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^cuResult$
+      - :name: ^pctx_val$
+  
+  - :set_id: primary_ctx_release
+    :name: "cuDevicePrimaryCtxRelease"
+    :register: false
+  - :set_id: primary_ctx_release_entry
+    :domain: entries & primary_ctx_release
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^dev$
+  - :set_id: primary_ctx_release_exit
+    :domain: exits & primary_ctx_release
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^cuResult$
+  
+  - :set_id: primary_ctx_reset
+    :name: "cuDevicePrimaryCtxReset"
+    :register: false
+  - :set_id: primary_ctx_reset_entry
+    :domain: entries & primary_ctx_reset
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^dev$
+  - :set_id: primary_ctx_reset_exit
+    :domain: exits & primary_ctx_reset
+    :payload_field_class:
+      :type: structure
+      :members:
+      - :name: ^cuResult$
+
+  # Stream create
+  - :set_id: stream_create
+    :name: "cuStreamCreate"
+    :register: false
+  - :set_id: stream_create_entry
+    :domain: entries & stream_create
+    :register: false
```

Comment:
Can you just delete it , if you don't need to register?

Reply:
yup kept it around "just in case" but clear now is not needed I think.